<ui version="4.0" >
 <class>SMTPInfoDlg</class>
 <widget class="QDialog" name="SMTPInfoDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>390</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>167772</width>
    <height>167772</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>SMTP Information</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="grpSMTPSettings" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>SMTP Mail Settings</string>
     </property>
     <widget class="QLabel" name="lebelHostIP" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>84</y>
        <width>235</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>235</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>SMTP Server's Host Name or IP</string>
      </property>
      <property name="text" >
       <string>Host Name/IP Address </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leHostIP" >
      <property name="geometry" >
       <rect>
        <x>151</x>
        <y>80</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>SMTP Server's Host Name or IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPortNumner" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>122</y>
        <width>216</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip" >
       <string>Port Number(Default Mail-server port is 25)</string>
      </property>
      <property name="text" >
       <string>Port Number </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPortNumber" >
      <property name="geometry" >
       <rect>
        <x>151</x>
        <y>120</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Port Number(Default is 25)</string>
      </property>
      <property name="maximum" >
       <number>65536</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblEncType" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>163</y>
        <width>216</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Mail Encryption Type</string>
      </property>
      <property name="text" >
       <string>Encryption Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbEncType" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>160</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>1000</width>
        <height>23</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Encryption Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbAuth" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>190</y>
        <width>130</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>1000</width>
        <height>23</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Authenticating Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="levelUsername" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>194</y>
        <width>240</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>230</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Authenticate with the Server</string>
      </property>
      <property name="text" >
       <string>Authentication Type </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUserName" >
      <property name="geometry" >
       <rect>
        <x>154</x>
        <y>230</y>
        <width>210</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Authenticating user name</string>
      </property>
     </widget>
     <widget class="QLabel" name="lebelPassword_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>233</y>
        <width>240</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>User name to be authenticate with the server</string>
      </property>
      <property name="text" >
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePassword" >
      <property name="geometry" >
       <rect>
        <x>151</x>
        <y>270</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>User authenticating password</string>
      </property>
      <property name="echoMode" >
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lebelPassword" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>273</y>
        <width>240</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>Password of the user to be authenticate with the server</string>
      </property>
      <property name="text" >
       <string>Password </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leProfileName" >
      <property name="geometry" >
       <rect>
        <x>151</x>
        <y>36</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>130</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>SMTP Server's Host Name or IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="lebelHostIP_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>235</width>
        <height>14</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>235</width>
        <height>14</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>240</width>
        <height>14</height>
       </size>
      </property>
      <property name="toolTip" >
       <string>SMTP Server's Host Name or IP</string>
      </property>
      <property name="text" >
       <string>Profile Name</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
